source/istio_virtualservice.go

```diff
@@ -186,6 +186,9 @@ func (sc *virtualServiceSource) endpointsFromVirtualService(ctx context.Context,
 		hosts = append(hosts, host)
 	}
 
+	// hosts contains dublicates here
```

Comment:
Add TODO string if the comment identifying things we should investigate

Reply:
done